### ISSUE TITLE
No images and weird font in default HTML preview

### ISSUE THREAD

**Issue by sirgazil** [retext-project/retext#239]
I'm using ReText 6.0.1 and the default HTML renderer uses a weird font for most of the text in every reSt document I try to preview.
## Steps to reproduce
1. Install ReText 6.0.1 from pypi.
2. Open ReText.
3. Write some reSt. For example:
   
   ```
   Casos de uso
   ============
   
   Ana Lyzer trabaja todo el día en el computador. Recientemente fue al
   médico quejándose de estrés y dolor de espalda. El médico le recomendó
   hacer pausas activas de 10 minutos cada hora.
   
   .. figure:: https://multimedialib.files.wordpress.com/2016/09/boceto-recordatorio.png
      :alt: Figura 1
      :align: center
   
      Figura 1. Boceto de cómo Ana quisiera que apareciera el recordatorio.
   ```
4. Press Ctrl+L to open the HTML preview.
## Unexpected behavior

The font used in paragraphs is weird, and the image is not displayed.

![Screenshot: weird font in ReText HTML preview](https://multimedialib.files.wordpress.com/2016/09/retext-html-preview-font-bug.png)
## Expected behavior

The font used in paragraphs is normal, and the image is displayed.

![Screenshot: normal font in ReText HTML preview with webkit](https://multimedialib.files.wordpress.com/2016/09/retext-html-preview-ok.png)
## Workaround

Use WebKit rendering for the preview by checking the "Edit → Use WebKit renderer" menu item.

**Comment by mitya57** [comment]
Thank you for your report.

Does this happen if you select another font using “Edit &rarr; Select preview font” menu item?

**Comment by sirgazil** [comment]
I opened "Edit → Select preview font", and the selected font was "Sans" (Regular), but I went ahead and clicked the OK button, and the font problem is gone.

However, the problem with images persists.

**Comment by mitya57** [comment]
I will look at the images problem later. Using WebKit usually gives a better experience anyway, so you can use it in the mean time.

**Comment by mitya57** [comment]
The problem with default fonts should be fixed in ReText 7.0.3, released in June (see commit 3e43d450d2f5554c06b1ab2f485e094d13b2add2).

With regards to images, the default renderer supports only local images, and I don't think we can do anything about that. You need to enable WebKit or WebEngine renderer to make external images work.
